app/Models/Topic.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Topic extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'topics';
+    /**
+     * [$dates description]
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+    /**
+     * [$fillable description]
+     * @var [type]
+     */
+    protected $fillable = [
+        'name',
+        'slug',
+        'images',
+        'set_time',
+        'select_time',
+        'status',
+        'created_at',
+        'updated_at',
+        'deleted_at'
+    ];
+    /**
+     * Get posts: One to many
+     * @return [type] [description]
+     */
+    public function posts() {
+        return $this->hasMany('App\Models\Post', 'topic_id');
+    }
+    /**
+     * Get users: Many to many
+     * @return [type] [description]
+     */
+    public function users() {
+        return $this->belongsTo('App\Models\User', 'user_id');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Topic.php</br>Opening brace should be on a new line